.eslintrc.js

```diff
@@ -0,0 +1,53 @@
+const _ = require('lodash');
+const path = require('path');
+const fs = require('fs')
+const yaml = require('js-yaml')
+const {execSync} = require('child_process');
+
+// Returns the path to the Indico source package/repo
+const PATH_COMMAND = "python -c 'from flask.helpers import get_root_path; print get_root_path(\"indico\")'";
+
+let indicoBaseDir = null;
+const indicoPathFile = path.join(__dirname, '.indico_source');
```

Comment:
out of curiosity, when can this folder be created?

Reply:
not sure if we are still using this for anything... @pferreir?